src/server/db_slice.cc

```diff
@@ -89,6 +89,14 @@ class PrimeEvictionPolicy {
   const bool can_evict_;
 };
 
+class PrimeBumpPolicy {
+public:
+  // returns true if key can be made less important for eviction (opposite of bump up)
+  bool CanBumpDown(const CompactObj& key) const {
+    return !key.IsSticky();
+  }
+};
+
```

Comment:
I've added a new policy. Pros: Easy to use. Cons: One more policy to manage.
But I guess you would've directly asked to just extend EvictionPolicy if you wanted me to.

Reply:
yeah, it's fine :)